octobot/src/server/github_handler.rs

```diff
@@ -11,7 +12,7 @@ use octobot_lib::config::Config;
 use octobot_lib::errors::Result;
 use octobot_lib::github;
 use octobot_lib::github::api::Session;
-use octobot_lib::github::CommentLike;
+use octobot_lib::github::{CommentLike, User};
```

Comment:
given this is is a large file that brings together github and jira things, i've tried to not import too many struct names directly. better to use `github::User` instead.

(the reason we need `CommentLike` imported here is because it's a trait and it needs to be imported before it can be used.)

```suggestion
use octobot_lib::github::CommentLike;
```